lms/urls.py

```diff
@@ -267,6 +267,11 @@
             'courseware.module_render.xblock_resource',
             name='xblock_resource_url'),
 
+        # xblock Rendering
+        url(r'^xblock/{usage_key_string}$'.format(usage_key_string=settings.USAGE_KEY_PATTERN),
```

Comment:
@nasthagiri @ormsbee Have we decided to move away from the ``/api/API_NAME/v0/...`` URL convention? If so, what is the new policy? What is the thinking regarding not including a version number?

It would be good to have this section of the guidelines updated if there are situations where we don't follow this practice:

https://openedx.atlassian.net/wiki/display/AC/edX+REST+API+Conventions#edXRESTAPIConventions-1.URLNaming